src/blocks/discord/base/set_bot_game.js

```diff
@@ -5,16 +5,56 @@ const blockName = "s4d_set_bot_game";
 
 const blockData = {
     "message0": "%{BKY_SET_BOT_GAME}",
-    "args0": [
-        {
+    "args0": [{
             "type": "input_value",
             "name": "GAME",
-            "check": [ "Number", "String" ]
+            "check": ["Number", "String"]
+        },
+        {
+            "type": "field_dropdown",
+            "name": "TYPE",
+            "options": [
+                [
+                    "%{BKY_LISTENING}",
+                    "LISTENING"
+                ],
+                [
+                    "%{BKY_WATCHING}",
+                    "WATCHING"
+                ],
+                [
+                    "%{BKY_COMPETING}",
+                    "COMPETING"
+                ],
+                [
+                    "%{BKY_PLAYING}",
+                    "PLAYING"
+                ]
+            ]
+        },
+        {
+            "type": "field_dropdown",
+            "name": "OIFD",
```

Comment:
why did you choose that name?

Reply:
as a game, meaning playing or listening. ok should I change it?
and I have to add invisible

Reply:
just have to add one label in fr pr and en and a very small edit in set bot game

Reply:
ok let me change it

Reply:
yes two embeds would not work.